public/docs/ts/latest/glossary.jade

```diff
@@ -250,7 +250,7 @@ a#decoration
     At the core, an [`injector`](#injector) returns dependency values on request.
     The expression `injector.get(token)` returns the value associated with the given token.
 
-    A token is an Angular type (`OpaqueToken`). You rarely need to work with tokens directly; most
+    A token is an Angular type (`InjectionToken`). You rarely need to work with tokens directly; most
```

Comment:
Actually, it provides type information that may be valuable but it provides almost zero type safety.